### ISSUE TITLE
Staging Review finding: Heading structure are siblings/same level but visually appear to be nested

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#139045]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Disability Benefits Crew Team 5 (12004)
**Product:** 21-526ez Disability Benefits Application
**Feature:** Reduce VA.gov-submitted Benefits Delivery at Discharge average days of claim (ADC) by requiring SHA Part A

## Findings details

**VA.gov Experience Standard - issue:** [User encounters a component or pattern that isn't considered a best practice.](https://design.va.gov/about/experience-standards/#user-encounters-components-and-patterns-that-meet-best-practices)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)

**Launch-blocking:** No 
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

"Submit your Separation Health Assessment" alert heading appears visually to be nested under ""Your Separation Health Assessment", however both headings are heading level 3, making them siblings. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="1057" height="357" alt=heading structure" src="https://github.com/user-attachments/assets/21164a50-88bb-4fb2-b2ef-54f0407a69c9" />

</details>

## Recommended action

Ensure the heading hierarchy matches the visual and structure of the page. 

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG22/#info-and-relationships)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by erynsobing** [comment]
Our team has some ongoing work to fix page headings and hierarchy across the whole 526 in preparation of replacing the forms segmented progress bar with a new Form Navigation component:
- #126024

Moved this issue under that epic, to be addressed as part of that effort.
